zellij-utils/assets/config/log4rs.yml

```diff
@@ -0,0 +1,42 @@
+# Scan this file for changes every 30 seconds
+refresh_rate: 30 seconds
+
+appenders:
+  # An appender named "stdout" that writes to stdout
+  stderr:
+    kind: console
+    target: stderr
+
+  logFile:
+    kind: file
+    path: "zellij.log"
+    append: false
+    encoder:
+      pattern: "{h(|{M}| {d} {l} [{T}] [{f}:{L}]: {m})} {n}"
```

Comment:
I should probably add comments explaining the patterns so everyone can see what the expected format is without going into log4rs documentation.

Reply:
Done, made the format more verbose so it should be clear how it should look like